app/src/main/res/values/dimens.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+  <!-- Story Fragment dimens. -->
+  <dimen name="padding_bottom">160dp</dimen>
+  <dimen name="custom_match_parent">-1px</dimen>
```

Comment:
@shivambh12 We cannot use negative values. If its not possible to merge all files without this solution then its fine to keep multiple files. Try to merge as many as you can without setting values in `negative`.

Also, please define all you dimens in a meaningful way (check names of all other values) and also they should be defined at the bottom on the xml file.

cc: @ARJUPTA

Reply:
@rt4914  I will define names of dimens meaningful way and also at bottom of xml.But, I tried to do with 0 dp(match_parent) instead of -1px but it doesn't work. So, I shouls leave two files one for phone and one for tablet.